[
    [
        "__str__",

        "(<slot wrapper '__str__' of 'unreal_engine.FAssetData' objects>, TypeError(\"descriptor '__str__' of 'unreal_engine.FAssetData' object needs an argument\",))"
    ],

    [
        "__init__",

        "(<slot wrapper '__init__' of 'unreal_engine.FAssetData' objects>, TypeError(\"descriptor '__init__' of 'unreal_engine.FAssetData' object needs an argument\",))"
    ],

    [
        "__new__",

        "(<built-in method __new__ of type object at 0x00007FFEE7367910>, TypeError('unreal_engine.FAssetData.__new__(): not enough arguments',))"
    ],

    [
        "get_asset",

        "(<method 'get_asset' of 'unreal_engine.FAssetData' objects>, TypeError(\"descriptor 'get_asset' of 'unreal_engine.FAssetData' object needs an argument\",))"
    ],

    [
        "is_asset_loaded",

        "(<method 'is_asset_loaded' of 'unreal_engine.FAssetData' objects>, TypeError(\"descriptor 'is_asset_loaded' of 'unreal_engine.FAssetData' object needs an argument\",))"
    ],

    [
        "get_thumbnail",

        "(<method 'get_thumbnail' of 'unreal_engine.FAssetData' objects>, TypeError(\"descriptor 'get_thumbnail' of 'unreal_engine.FAssetData' object needs an argument\",))"
    ],

    [
        "has_custom_thumbnail",

        "(<method 'has_custom_thumbnail' of 'unreal_engine.FAssetData' objects>, TypeError(\"descriptor 'has_custom_thumbnail' of 'unreal_engine.FAssetData' object needs an argument\",))"
    ],

    [
        "has_cached_thumbnail",

        "(<method 'has_cached_thumbnail' of 'unreal_engine.FAssetData' objects>, TypeError(\"descriptor 'has_cached_thumbnail' of 'unreal_engine.FAssetData' object needs an argument\",))"
    ],

    [
        "asset_class",

        "(<attribute 'asset_class' of 'unreal_engine.FAssetData' objects>, TypeError(\"'getset_descriptor' object is not callable\",))"
    ],

    [
        "asset_name",

        "(<attribute 'asset_name' of 'unreal_engine.FAssetData' objects>, TypeError(\"'getset_descriptor' object is not callable\",))"
    ],

    [
        "object_path",

        "(<attribute 'object_path' of 'unreal_engine.FAssetData' objects>, TypeError(\"'getset_descriptor' object is not callable\",))"
    ],

    [
        "package_flags",

        "(<attribute 'package_flags' of 'unreal_engine.FAssetData' objects>, TypeError(\"'getset_descriptor' object is not callable\",))"
    ],

    [
        "package_name",

        "(<attribute 'package_name' of 'unreal_engine.FAssetData' objects>, TypeError(\"'getset_descriptor' object is not callable\",))"
    ],

    [
        "package_path",

        "(<attribute 'package_path' of 'unreal_engine.FAssetData' objects>, TypeError(\"'getset_descriptor' object is not callable\",))"
    ],

    [
        "tags_and_values",

        "(<attribute 'tags_and_values' of 'unreal_engine.FAssetData' objects>, TypeError(\"'getset_descriptor' object is not callable\",))"
    ],

    [
        "__doc__",

        "('Unreal Engine FAssetData', TypeError(\"'str' object is not callable\",))"
    ]
]